### DISCUSSION TITLE
Disable cursor disappearing on rewind bar

### DISCUSSION THREAD

**Discussion by nlboom** [smplayer-dev/smplayer#113] [Ideas]
It often happens that in the film you need to find the desired scene, and you move the cursor on rewind bar quite a bit, so as not to miss anything, and cursor disappears. This is very pedestrian.

**Comment by new-penguin** [comment]
Maybe this should be labeled a bug. Happens to me also on 23.6.0-1 on Arch w/ Wayland, AMDGPU and it's a bit annoying.

**Reply by Raza-0007** [reply]
It must be platform specific. I have not experienced it on Windows 10 in my 5+ years of using smplayer.

**Reply by new-penguin** [reply]
I should clarify it just does this in compact mode while windowed.

**Reply by Raza-0007** [reply]
I see......maybe this is why I have never experienced it as I have never used compact mode before. 

I just tested the compact mode, both in windowed and full screen, and could not reproduce the issue. When you say cursor, I am assuming you mean the progress bar that displays on screen, that you can move backwards and forwards with keyboard shortcut. It did not disappear for me. Unless I am misunderstanding your issue.

Or it could be GUI dependent. Under Option>Preferences>Interface> I am on the "basic GUI", with default icon set, and "fusion" style.

**Reply by new-penguin** [reply]
Yes, the cursor disappears when it's over any part of the menu bar in compact windowed mode, be it on the progress bar or the play, forward, back icons. Of course you'd normally expect it to hide when only over the video, not the interactive interface. So far it's my only issue with it as it's my favorite media player on Linux.

**Reply by Raza-0007** [reply]
I see.... so you are talking about the mouse cursor. By the way, how do you have menu and progress bars on your windowed compact mode? A windowed compact mode should remove everything from the screen but the playing video. 

I do see the bottom progress bar when I put the compact mode in full screen, and the mouse cursor does disappear on it after a while, but I think this is by design, and not a bug.  The whole point of the compact mode is that it removes all the distractions, so all you see is the video on the screen, and nothing else.

~Raza

**Reply by new-penguin** [reply]
When you mouse-over where the menu is, it pops up in compact mode. Otherwise, yes, the mouse cursor does disappear as expected everywhere else on the video.

**Reply by Raza-0007** [reply]
@new-penguin, your issue must be platform dependent. On Win 10, when I put smplayer in a windowed compact mode from `Video>Compact mode` or `Ctrl+C`, I am then unable to see any menu or progress bars. Nothing pops out if I move my mouse around, so I am unable to replicate the issue you mentioned. Perhaps someone on Arch Linux can test it to see if they are able to reproduce it.

**Reply by new-penguin** [reply]
When I move my mouse in compact mode when playing a media file, nothing happens unless I move my mouse to the bottom of the window and then the progress bar and such appears. That's what I'd expect. What OP and I are talking about is the mouse cursor disappears when hovering over the progress bar when the progress bar appears in compact mode. Sorry for the misunderstanding, I think I used "menu bar" instead of progress bar before, maybe that led to some confusion.

**Reply by Raza-0007** [reply]
@new-penguin, watch this screenshot sample video (without audio) inside the zip file. I am unable to replicate the progress bar appearing on a windowed compact mode.

[Sample Video.zip](https://github.com/smplayer-dev/smplayer/files/12480247/Sample.Video.zip)

**Reply by new-penguin** [reply]
Maybe you should file a bug report but it works for me. It used to behave that way a few months ago on an older version. What doesn't work is the mouse staying present when hovering over the progress bar when it appears. I'm not sure why you're replying to me, you seem to have a different issue.

**Reply by Raza-0007** [reply]
I was trying to resolve your issue!

I see you do not need resolution. I will stop wasting my time then. 

Bye.
